src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts

```diff
@@ -496,6 +508,34 @@ export const useLoadingChart = (props: LoadingChartHookProps) => {
         }
     }, [throttledInViewportCheck]);
 
+    /**
+     * reload chart by timer when the _autoupdate param is passed
+     */
+    const reloadChart = React.useCallback(() => {
+        const autoupdateIntervalMs = Number(autoupdateInterval) * 1000;
+        if (autoupdateIntervalMs) {
+            const timeSinceLastReload = new Date().getTime() - (lastReloadAt || 0);
+            const reloadIntervalRemains = autoupdateIntervalMs - timeSinceLastReload;
+
+            if (
+                (!isPageHiddenCurrent || isEmbeddedMode()) &&
+                reloadIntervalRemains <= 0 &&
+                reloadTimeout
+            ) {
+                lastReloadAt = new Date().getTime();
+                loadChartData();
+            }
+
+            reloadTimeout = setTimeout(() => reloadChart(), autoupdateIntervalMs);
+        }
+    }, [loadChartData, autoupdateInterval]);
+
+    // need update timeout if it's first render with autoupdateInterval = 0, or if visibility of page is changed
+    React.useEffect(() => {
+        clearTimeout(reloadTimeout);
+        reloadChart();
+    }, [isPageHidden, autoupdateInterval]);
```

Comment:
I suppose you need `reloadChart` in deps

Reply:
After the experiments, we decided to leave it as it is